relay-redis/src/real.rs

```diff
@@ -30,50 +35,109 @@ pub struct Connection<'a> {
     inner: ConnectionInner<'a>,
 }
 
-impl ConnectionLike for Connection<'_> {
-    fn req_packed_command(&mut self, cmd: &[u8]) -> redis::RedisResult<redis::Value> {
-        match self.inner {
+impl Connection<'_> {
+    fn req_packed_command_inner(
+        inner: &mut ConnectionInner<'_>,
```

Comment:
I think implementing this on the `ConnectionInner` would make it a bit nicer, it allows you to just `primary.req_packed_command_inner`, since these functions are essentially just methods. Might as well implement the trait on the `ConnectionInner` and then just forward all calls.